charts/icm-as/docs/how-to.asciidoc

```diff
@@ -0,0 +1,56 @@
+= What has to be done to reach a certain goal
```

Comment:
Strange title. Are the two sections below related to each other at all? If not, maybe split into two docs with suitable titles?

Reply:
The how-to.asciidoc is comparable to our cookbooks. Feel free to propose another title.